refarch-integrations/refarch-s3-integration/refarch-s3-integration-service/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- The possibly existing properties concerning the "logging.level" in the application.yml
+     have priority over the log level maintained in this file.  -->
+<configuration>
+
+
+    <springProperty scope="context" name="springAppName" source="spring.application.name"/>
+
+
+    <springProfile name="!json-logger">
```

Comment:
Should we also rename this to `json-logging` as this is the default we have in the gateway as well?